Cas 391/postpone query channels (#866)
GetStream/stream-chat-android: CHANGELOG.md

## stream-chat-android

## stream-chat-android-client

> - There is no more the requirement to wait set user before querying channels

Reviewer: ```suggestion
- It's no longer required to wait for `setUser` to finish before querying channels
```
